L1Trigger/Phase2L1ParticleFlow/plugins/L1CTJetFileWriter.cc

```diff
@@ -101,12 +142,33 @@ std::vector<ap_uint<64>> L1CTJetFileWriter::encodeJets(const std::vector<l1t::PF
   return jet_words;
 }
 
+std::vector<ap_uint<64>> L1CTJetFileWriter::encodeSums(const std::vector<l1t::EtSum> sums) {
+  // Send MHT first, then MET
+  // Need two l1t::EtSum for each MHT, MET (four total)
+  // No MET consumed for now - send 0s on second word
+  std::vector<ap_uint<64>> sum_words;
+  int valid = sums.at(0).pt() > 0;
+  // TODO this is not going to be bit exact
```

Comment:
why not? you're just copying around bits that have already been digitized, so there shouldn't be any inaccuracy here

Reply:
This comment was just a relic from the old code, the code in this PR is bit exact. I removed the comment